webplatform/config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Webplatform::Application.routes.draw do
   root 'home#index'
+  get 'subscribe' => 'subscribe#index'
+  post 'subscribe/subscribe' => 'subscribe#subscribe'
```

Comment:
By default Rails creates routes for the seven default actions (index, show, new, create, edit, update, and destroy). And you can use the :only and :except options to restrict which actions you're going to use so in this case maybe instead of creating method subscribe which basically create new subscription let's use create method and then in your routes you can only add `resources :subscribes, only: [:index, :create]`